README.md

```diff
@@ -173,6 +173,11 @@ Default is `0` (no auto-show behavior).
 If enabled the buffers will be displayed in a reversed order.
 Default is `0` (buffers are not reversed).
 
+##### `g:lightline#bufferline#reverse_buffer_icons`
+
+If set to `1`, buffer icons are displayed on the right side of the file name.
+If `0`, it will be displayed on the left side. Default is `0`.
+
```

Comment:
```suggestion
##### `g:lightline#bufferline#icon_position`

 If set to `right`, filetype icons are displayed on the right side of the file name.
 If `left` or omitted, it will be displayed on the left side. Default is `left`.
 
```